cmake/configure_wcoss2.cmake

```diff
@@ -0,0 +1,3 @@
+set(INLINE_POST     ON  CACHE BOOL "Enable inline post" FORCE)
+set(PARALLEL_NETCDF ON  CACHE BOOL "Enable parallel NetCDF" FORCE)
+set(DEBUG_LINKMPI   OFF CACHE BOOL "Enable linkmpi option when DEBUG mode is on" FORCE)
```

Comment:
Are we debugging MPI implementation here?

I see we are hard-wiring flags such as:

```
if(DEBUG)
     if(OPENMP)
        set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -link_mpi=dbg_mt")
      else()
        set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -link_mpi=dbg")
      endif()
endif()
```

This should be handled by linking to the runtime debug MPI library by setting an environment variable such as `I_MPI_LINK`

By adding those flags at compile time, we are essentially debugging the MPI implementation.